mParticle-BranchMetrics.podspec

```diff
@@ -16,5 +16,5 @@ Pod::Spec.new do |s|
     s.ios.deployment_target = "9.0"
     s.ios.source_files      = 'mParticle-BranchMetrics/*.{h,m,mm}'
     s.ios.dependency 'mParticle-Apple-SDK/mParticle', '~> 8.2'
-    s.ios.dependency 'Branch', '~> 1.39.1'
+    s.ios.dependency 'Branch', '~> 1.39.4'
```

Comment:
Based off this
```
Note: Requires Branch iOS SDK version 1.39.4 or higher, but I've implemented the method call in our kit wrapper to fail gracefully if using an older version of their SDK.
```
I figured you wouldn't want to update the version to require 1.39.4 ?

Reply:
So when I looked cloned their repo and checked the latest in their master branch, the `checkPasteboardOnInstall` was nowhere to be found in their code for some reason, even though master is ahead of the 1.39.4 release which _does_ have that method... So I figured I'd be extra save when calling it in case they release some future version that doesn't include the method. Then I know of cases (which we don't officially support), where mParticle clients use our kit but manually include the partner framework, so this handles that hypothetical case as well of someone pinning an older partner SDK but for some reason updating the kit.

Reply:
Updating my note in the PR description to mention this.